Examples_Nuget/quickstart_cloud/Program.cs

```diff
@@ -0,0 +1,154 @@
+﻿ 
+using TileDB;
+using System;
+
+
+
+//Login
+string host = "https://api.tiledb.com"; //default host
+string api_key = System.Environment.GetEnvironmentVariable("REST_TOKEN_KEY"); //default token
+TileDB.Cloud.Client.Login(token:api_key,host:host);
+//TileDB.Cloud.Client.Login(); // use default values
+
+// Get User information
+var user_details = TileDB.Cloud.RestUtil.GetUser();
+System.Console.WriteLine("{0}", user_details.ToJson());
+
+//Array uri infos
+string original_file = "s3://tiledb-inc-demo-data/files/original/VLDB17_TileDB.pdf";
+
+//TileDB File details
+string tiledb_file_name = "VLDB17_TileDB_Example";
+string tiledb_file_s3_uri = user_details.DefaultS3Path + "/" + tiledb_file_name;
+System.Console.WriteLine("tiledb_file_s3_uri:{0}", tiledb_file_s3_uri);
+
+//Delete the file array if exists
```

Comment:
Comment this out or change the URI so that we don't accidentally delete the original?